docs/manual/interpreters.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: "Interpreters"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+
+## Interpreters in zeppelin
+
+This section explain the role of Interpreters, interpreters group and interpreters settings in Zeppelin.
+Zeppelin interpreter concept allows any language/data-processing-backend to be plugged into Zeppelin.
+Currently Zeppelin supports many interpreters such as Scala(with Apache Spark), Python(with Apache Spark), SparkSQL, Hive, Markdown and Shell.
+
+### What is zeppelin interpreter?
+
+Zeppelin Interpreter is the plug-in which enable zeppelin user to use a specific language/data-processing-backend. For example to use scala code in Zeppelin, you need ```spark``` interpreter.
+
+When you click on the ```+Create``` button in the interpreter page the interpreter drop-down list box will present all the available interpreters on your server.
+
+<img src="../../assets/themes/zeppelin/img/screenshots/interpreter_create.png">
+
+### What is zeppelin interpreter setting?
+
+Zeppelin interpreter setting is the configuration of a given interpreter on zeppelin server. For example, the properties requried for hive  JDBC interpreter to connect to the Hive server.
+
+<img src="../../assets/themes/zeppelin/img/screenshots/interpreter_setting.png">
+### What is zeppelin interpreter group?
+
+Every Interpreter belongs to an InterpreterGroup. InterpreterGroup is a unit of start/stop interpreter.
+By default, every interpreter belong to a single group but the group might contain more interpreters. For example, spark interpreter group include spark support, pySpark, 
+SparkSQL and the dependency loader.
+
+Technically, Zeppelin interpreters from the same group are running in the same JVM.
+
+**TODO: Validate this with the community ...**
```

Comment:
I wasn't sure that the description of what is a registered interpreter,
setting and group mean.
If it is correct I will remove the TODO and the url parameter as commented.
BTW, Internally, I think the doing a create setting which is actualky
calling add group is a bed naming and hard to follow.
Has no effect on documentation for now...

בתאריך יום ו׳, 28 באוג׳ 2015, 09:05 מאת Alexander <notifications@github.com
>:

> In docs/manual/interpreters.md
> <https://github.com/apache/incubator-zeppelin/pull/256#discussion_r38173203>
> :
>
> > +<img src="../../assets/themes/zeppelin/img/screenshots/interpreter_create.png">
> > +
> > +### What is zeppelin interpreter setting?
> > +
> > +Zeppelin interpreter setting is the configuration of a given interpreter on zeppelin server. For example, the properties requried for hive  JDBC interpreter to connect to the Hive server.
> > +
> > +<img src="../../assets/themes/zeppelin/img/screenshots/interpreter_setting.png">
> > +### What is zeppelin interpreter group?
> > +
> > +Every Interpreter belongs to an InterpreterGroup. InterpreterGroup is a unit of start/stop interpreter.
> > +By default, every interpreter belong to a single group but the group might contain more interpreters. For example, spark interpreter group include spark support, pySpark,
> > +SparkSQL and the dependency loader.
> > +
> > +Technically, Zeppelin interpreters from the same group are running in the same JVM.
> > +
> > +**TODO: Validate this with the community ...**
>
> Do we want to keep if for now, or you need some kind of input to resolve
> it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/256/files#r38173203>.
>